packages/modules/itwin-viewer-react/src/components/app-ui/widgets/BasicNavigationWidget.tsx

```diff
@@ -33,16 +33,69 @@ export function BasicNavigationWidget({ config }: BasicNavigationWidgetProps) {
     if (config?.hideDefaultHorizontalItems) {
       return [];
     }
-    const items: CommonToolbarItem[] = ToolbarHelper.createToolbarItemsFromItemDefs(
-      [
-        CoreTools.rotateViewCommand,
-        CoreTools.panViewCommand,
-        CoreTools.fitViewCommand,
-        CoreTools.windowAreaCommand,
-        CoreTools.viewUndoCommand,
-        CoreTools.viewRedoCommand,
-      ]
-    );
+    const items: CommonToolbarItem[] = [];
+
+    if (
```

Comment:
just to note, users can still rotate & pan using their keyboard+mouse which is nice